sbncode/LArRecoProducer/ShowerCosmicCylinder_module.cc

```diff
@@ -0,0 +1,170 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       ShowerCosmicCylinder
+// Plugin Type: producer (art v3_03_01)
+// File:        ShowerCosmicCylinder_module.cc
+//
+// Generated at Tue Sep 15 08:54:39 2020 by Edward Tyley using cetskelgen
+// from cetlib version v3_08_00.
+//
+// Producer to find the distance of closest approach from a shower to
```

Comment:
Ah. So now I know what this number is. Can we call the CAF variable `shw.cosmicdist` or something then?